### PR TITLE
fixed date-fns imports

### PR DESCRIPTION
**Pull request by hoesele**
See Issue https://github.com/xmtp/xmtp-web/issues/124 for details.

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -5,3 +5,5 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
     spec: "@yarnpkg/plugin-interactive-tools"
+
+yarnPath: .yarn/releases/yarn-3.6.4.cjs
```
Comment (neekolas) [inline .yarnrc.yml:9]: Is this change necessary for the fix?

Comment (hoesele) [reply .yarnrc.yml:9]: no, sorry only the changed imports of date-fns

**Comment by hoesele** [comment]
@neekolas: I've removed the unnecessary files from the commit

File: packages/react-sdk/src/helpers/caching/contentTypes/readReceipt.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import {
 import { ContentTypeId } from "@xmtp/xmtp-js";
 import { parseISO } from "date-fns";
 import { z } from "zod";
-import isAfter from "date-fns/isAfter";
+import { isAfter } from "date-fns";
```
Comment (rygine) [inline packages/react-sdk/src/helpers/caching/contentTypes/readReceipt.ts:8]: the linter requires that there's only 1 import statement per dependency. please combine the `isAfter` and `parseISO` imports into 1 statement.

File: .changeset/loud-brooms-sparkle.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+"@xmtp/react-sdk": major
```
Comment (rygine) [inline .changeset/loud-brooms-sparkle.md:2]: this should be a `patch` release instead of `major`

**Review by rygine — APPROVED** [review]

**Comment by rygine** [comment]
looks great, thanks again for the PR!
